nautobot/core/testing/views.py

```diff
@@ -1321,12 +1320,11 @@ def test_bulk_delete_form_contains_all_filtered(self):
             # Expect a 200 status cause we are only rendering the bulk delete table after pressing Delete Selected button.
             self.assertHttpStatus(response, 200)
             response_body = utils.extract_page_body(response.content.decode(response.charset))
-            # Check if the first and second pk is passed into the form.
-            self.assertIn(f'<input type="hidden" name="pk" value="{first_pk}"', response_body)
-            self.assertIn(f'<input type="hidden" name="pk" value="{second_pk}"', response_body)
-            self.assertIn("<strong>Warning:</strong> The following operation will delete 2 ", response_body)
-            # Check if the third pk is not passed into the form.
+            # Check if all pks is not part of the html.
+            self.assertNotIn(f'<input type="hidden" name="pk" value="{first_pk}"', response_body)
+            self.assertNotIn(f'<input type="hidden" name="pk" value="{second_pk}"', response_body)
             self.assertNotIn(f'<input type="hidden" name="pk" value="{third_pk}"', response_body)
+            self.assertIn("<strong>Warning:</strong> The following operation will delete 2 ", response_body)
```

Comment:
Might be worth to use `assertNotInHtml` and `assertInHtml` just in case that formatting change in template don't break the tests.

Reply:
There is no `assertNotInHtml`, also the full html tag is a bit unpredictable `<input type="hidden" name="pk" value="<uuid>" class="form-control" required="required" placeholder="None" id="id_pk_0">` as I dont really know which  might be `id__pk__1` or `id__pk__2` etc.

Reply:
Can possibly use `assertNotContains(..., html=True)`.